### DISCUSSION TITLE
po-TableColumn - type:currency

### DISCUSSION THREAD

**Discussion by egolob** [po-ui/po-angular#991] [Angular]
Ola pessoal.

Ao utilizar o poTableColumn e definir uma propriedade de type:currency, é possivel formatar as casas decimais?

Não encontrei uma forma. Consigo formatar as casa apenas com type:number.

Att

**Comment by jhonyeduardo** [comment]
Boa tarde @egolob 

atualmente não, teria que abrir uma issue para permitir.
